lib/core.js

```diff
@@ -747,3 +733,21 @@ impress.logApiMethod = fnPath => {
     originalMethod(...args);
   });
 };
+
+impress.startTests = () => {
+  api.test.runner.instance.resume();
+  api.test.test('Impress applications test', test => {
+    test.endAfterSubtests();
+    Object.values(impress.applications).forEach(app => {
+      test.test(app.name, test => {
+        test.endAfterSubtests();
+        app.tests.unit
+          .concat(app.tests.integration)
+          .forEach(testCase => test.test(testCase.path, testCase.test));
+      });
+    });
+  });
+  api.test.runner.instance.on('finish', hasErrors => {
+    process.send({ name: 'impress:testsFinished', hasErrors });
```

Comment:
Shouldn't we wait for some time in case if there are tests that can fail after the `finish` event?